ceph_installer/controllers/mon.py

```diff
@@ -54,14 +54,17 @@ def configure(self):
     @configure.when(method='POST', template='json')
     @validate(schemas.mon_configure_schema, handler="/errors/schema")
     def configure_post(self):
-        hosts = [request.json['host']]
+        hosts = ["{} monitor_interface={}".format(request.json['host'], request.json["monitor_interface"])]
```

Comment:
this looks very familiar to what `parse_monitors` does, why don't just use that?